### ISSUE TITLE
Rule proposal: sorted propTypes

### ISSUE THREAD

**Issue by lencioni** [yannickcr/eslint-plugin-react#38]
Similar to #16, some people might might to enforce that propTypes are sorted alphabetically.

Good:

```jsx
propTypes: {
  aProp: React.PropTypes.object,
  bProp: React.PropTypes.object,
  cProp: React.PropTypes.object,
},
```

Bad:

```jsx
propTypes: {
  bProp: React.PropTypes.object,
  aProp: React.PropTypes.object,
  cProp: React.PropTypes.object,
},
```
